trlx/model/accelerate_base_model.py

```diff
@@ -140,6 +140,10 @@ def save(self, directory=None):
         """Creates checkpoint of optimizer, scheduler and a model"""
         self.accelerator.save_state(directory or self.config.train.checkpoint_dir)
 
+    def load(self, directory=None):
+        """load checkpoint of optimizer, scheduler and a model"""
+        self.accelerator.load_state(directory or self.config.train.checkpoint_dir)
```

Comment:
Delete the `get_components` override here: https://github.com/CarperAI/trlx/blob/ee6e58c5417ca56e0cb93a1eb701b61889f0c6e4/trlx/model/accelerate_base_model.py#L131-L138
since it will never get called after your `load` update.